themes/hugo-lithium/static/css/main.css

```diff
@@ -366,3 +366,14 @@ code {
     box-shadow: none;
   }
 }
+
+
+/* bring quotes coser to previous paragraph */
+blockquote {
+  margin-block-start: -1em;
```

Comment:
This feels a little too close now to my eye... also, given that the inter-para spacing is set by pixels and not ems, this is inconsistent when you resize. What about something like `margin-block-start: -10px;` ?

Reply:
The theme mixes em and px, but yeah the main spacing for paragraphs is `25px`. I've tweaked it a bit so it's clear that the bullets and blockquotes are connected to the preceding paragraph but aren't smashed into it.

Reply:
it still looks a little close to me... if your eye says -15px and mine says -10px, maybe we compromise on -12.5px? That also happens to be have the inter-para spacing, so maybe that's good?

(also you set the bullet spacing to be -19px, which definitely looks too close to me. Whatever we set this to I would make them both the same)

Reply:
OK. I set both to -12.5px

Reply:
and preview site's updated